robot/Crisis-Management/tests/browser/t3_availability_records_created.robot

```diff
@@ -0,0 +1,59 @@
+*** Settings ***
+
+Resource        robot/Crisis-Management/resources/CrisisManagement.robot
+Library
+...             cumulusci.robotframework.PageObjects
+...             robot/Crisis-Management/resources/ContactPageObject.py
+...             robot/Crisis-Management/resources/CrisisManagement.py
+
+Suite Setup     Run keywords
+...             Open Test Browser
+...             Setup Test Data
+
+Suite Teardown  Delete Records and Close Browser
+
+***Keywords***
+
+# Set up all the required data for the test based on the keyword requests
+Setup Test Data
+    &{main_account} =                  API Create Account  Regional_Health_Authority   Name=Automtion Health Services
+    &{hospital} =                      API Create Account  Hospital                    Name=Robot_test Hospital        ParentId=&{main_account}[Id]
+    Set Suite Variable                 &{hospital}
+
+*** Variables ***
+${status1}    On staff
+${status2}    Not Available
+
+
+*** Test Cases ***
+
+1. Create a new staff with status equals on staff
+   # Login as admin and create a new staff wit status Not Available
+    Go To Page                                           Listing                                 Contact
+    Click Object Button                                  New
+    Wait For Modal                                       New                                     Staff
+    Populate Modal Form
+    ...                                                  First Name=${faker.first_name()}
+    ...                                                  Last Name=${faker.last_name()}
+    ...                                                  Status=${status1}
+    Populate Lookup Field                                Account Name                            &{hospital}[Name]
```

Comment:
```suggestion
    Populate Lookup Field                                Facility Name                            &{hospital}[Name]
```

I think there might have been a recent change by the traction team to the field name. Test was failing for me with Account but works with Facility.